lib/commands/serve.js

```diff
@@ -32,7 +34,7 @@ class ServeCommand extends Command {
     const { builds = [], routes = [] } = require(nowConfigPath)
 
     // find all node lambdas
-    const nodeBuilds = builds.filter(({ use }) => use === '@now/node')
+    const nodeBuilds = builds.filter(({ use }) => /^@now\/node@?/.test(use))
```

Comment:
At the moment, Typescript support on Now v2 is only possible via the canary version of `@now/node`, hence the check for the optional `@` (i.e. `@now/node@canary` will be the builder for some time)